docs/integrations/database-access-control/trino-integration.mdx

```diff
@@ -0,0 +1,365 @@
+---
+sidebar_position: 1
+title: Trino Integration
+description: Use Permit to enforce fine-grained authorization decisions for every query that flows through Trino.
+---
+
+## Overview
+
+Trino is an open-source distributed SQL engine that sits in front of the data you store in your databases.
+Because all queries route through a single coordinator, Trino becomes an ideal point for enforcing database-level access control.
+You can learn more about Trino in their [official documentation](https://trino.io/docs/current/).
+
+Pairing Trino with Permit's PDP enables you to:
+
+- Centralize database-level permission logic with Permit’s Policy Builder.
+- Control access to databases and tables, and even individual columns.
+- Filter or mask data before it leaves your databases, based on user permissions.
+- Enforce the decision before data leaves your database.
+- Audit access to your data using Permit's audit logs.
+
+The integration is most effective for organizations that want to enforce database-level access control across all their databases.
+It can apply both to internal users, like analysts accessing data using BI tools, and external users accessing data through APIs
+or other applications.
+
+## Quick Start
+
+1. **Prepare Permit**
+   - Create a new [Permit environment](/manage-your-account/projects-and-env#creating-a-new-environment) dedicated to the Trino cluster.
+   - Copy the PDP API key from the dashboard.
+2. **Sync the schema**
+   - Install the latest [Permit CLI](/how-to/permit-cli).
+   - Run `permit env apply trino --url http://<trino-host>:8080 --user <trino-user>` and point it at a user that can read cluster metadata.
+   - The CLI introspects catalogs, schemas, tables, functions, and (optionally) columns, then creates matching resources and actions in Permit.
+3. **Deploy the PDP**
+   - Run a Permit PDP that targets the same environment. Ensure Trino can reach it (default port `7766`).
+   - Enable `PDP_ALLOW_UNAUTHENTICATED_TRINO=True` to allow unauthenticated Trino queries to be checked by the PDP.
+4. **Configure Trino to use the PDP for access control**
+   - Configure the Trino's `/etc/trino/access-control.properties` to call the PDP:
+   ```yaml
+   access-control.name=opa
+   opa.policy.uri=http://<pdp-host>:7766/trino/allowed
+   opa.policy.row-filters-uri=http://<pdp-host>:7766/trino/row-filter
+   opa.policy.batch-column-masking-uri=http://<pdp-host>:7766/trino/batch-column-masking
+   opa.log-requests=true
+   opa.log-responses=true
+   ```
+   - Restart the Trino coordinator.
```

Comment:
Maybe add a link to the Trino PDP config - i.e. [optional] configure PDP setting for Trino (via /app/config/trino-authz.yaml)